internal/locale/locale.go

```diff
@@ -155,8 +156,13 @@ func Tt(translationID string, args ...interface{}) string {
 
 	// For templates we want to manually specify the linebreaks as the way YAML gets parsed makes
 	// this very painful otherwise
-	translation = strings.Replace(translation, "\n", "", -1)
+
+	// Replace newlines in yaml strings with space to avoid concatenated words
+	replaceRegex := regexp.MustCompile(`[\ ]?\n(\w|\{)`)
```

Comment:
think we could simplify this to `s/\s*\n/ /`

Reply:
I've updated this to `\s*\n` which works in my testings. `/s` matches to the characters `/s`, same with `/ /`.